apis/go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/openshift/api v3.9.0+incompatible // indirect
 	github.com/openstack-k8s-operators/lib-common/modules/openstack v0.0.0-20220915080953-f73a201a1da6 // indirect
 	github.com/openstack-k8s-operators/lib-common/modules/storage v0.0.0-20220923094431-9fca0c85a9dc // indirect
+	github.com/openstack-k8s-operators/ovs-operator/api v0.0.0-20221201145301-12f408899b26
```

Comment:
let's also update to latest https://github.com/openstack-k8s-operators/ovs-operator/commits/main

Reply:
Done